docs/blog/2020-community-update.rst

```diff
@@ -0,0 +1,115 @@
+
+We hope that this data can help our community members determine what appropriate salary ranges are, and provide a benchmark for future employment-related decisions and negotiations.
+Additionally, we’re hoping the results spark discussion of employment-related trends and issues in our industry, including but not limited to the effects of the COVID-19 pandemic.
+
+The survey is broken up into 5 sections.
+It should take between 5 and 10 minutes to complete,
+so we hope that you'll `fill it out <https://2020-salary-survey.writethedocs.org/>`_.
+
+Community Shout Out
+-------------------
+
+The numbers of activities going on in the Write the Docs community grows each year.
+Most of these are things are organized and run primarily by volunteers,
+and we wanted to take time to show appreciation for these folks.
+We will certainly miss many of people doing great work around out community,
+but we wanted to show appreciation for folks in particular who are working behind the scenes.
+
+Newsletter Team
+~~~~~~~~~~~~~~~
+
+First up, as seems fitting, is our Newsletter team.
+Y'all probably know Beth,
+who is our editor,
+but there is an entire group working behind the scenes to write the articles each month.
+They are responsible for making this newsletter happen each month:
+
+* `Beth Aitman <https://twitter.com/baitman>`_
+* `Hillary Fraley <https://github.com/hillaryfraley>`_
+* `Jennifer Rondeau <https://twitter.com/bradamante>`_
+* `Claire Lundeby <https://twitter.com/clairelundeby>`_
+* `Elle Jones <https://twitter.com/party_parrot18>`_
+* `Kyla del Rosario <https://kyladelrosario.com/>`_
+
+Moderation Team
+~~~~~~~~~~~~~~~
+
+Next is our Slack moderators.
+They do a ton of work to make sure the Slack network stays a welcoming place and people feel good about interacting:
+
+* `Jennifer Rondeau <https://twitter.com/bradamante>`_
+* `Janine Chan <https://www.linkedin.com/in/janinechan/>`_
+* `Daniel Beck <https://twitter.com/ddbeck>`_
+* `Mike Jang <https://twitter.com/themikejang>`__
+* `Eric Holscher <https://twitter.com/ericholscher>`_
+* `Samuel Wright <https://twitter.com/plaindocs>`_
+
+Meetup Coordination Team
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are too many meetup organizers to list across our lovely community.
+Y'all get to know them at each meetup that you go to,
+but behind the scenes there are folks who are doing coordination to make our meetup community stronger.
+That is the job of the meetup coordination team,
+and our meetups wouldn't be as useful without them:
+
+* `Rose Williams <https://twitter.com/ZelWms>`_
+* `Margaret Eker <https://twitter.com/meker>`_
+* `Mike Jang <https://twitter.com/TheMikeJang>`_
+* `Swapnil Ogale <https://twitter.com/swapnilogale>`_ (APAC)
```

Comment:
Seems odd to put the APAC here

Reply:
Yea, just copied from the team page, removed.